pftp/data_handler_test.go

```diff
@@ -301,14 +301,44 @@ func Test_dataHandler_parsePASVresponse(t *testing.T) {
 			wantErr: true,
 		},
 		{
-			name: "passive_mode_parse_ok",
+			name: "passive_mode_parse_ok_public",
 			fields: fields{
-				line:   "227 Entering Passive Mode (10,19,10,10,100,10).\r\n",
+				line:   "227 Entering Passive Mode (20,30,40,50,100,10).\r\n",
```

Comment:
Why did you need to change an existing test?

Reply:
Because the existing one didn't work: https://github.com/pyama86/pftp/blob/master/pftp/data_handler_test.go#L306 We sent `10,19,10,10` (< typo 19 instead of 10) and checked for `10.10.10.10`. Also as we ignore private IPs by default it was not used.

The logic of the existing test is now called `passive_mode_parse_ok_private`

Reply:
I see.
To be honestry,I don't use this software myself for a long time.
However,my colleagues continue using and I shoud maintain this repository.
I habe added GitHub Actions to in this repository and I hope  keep quority this repository.
thanks for you contribute.

Reply:
could you please pull origin master!

Reply:
done